nomination/templates/nomination/help.html

```diff
@@ -1,78 +1,161 @@
-<h3>CAN I DELETE ENTRIES?</h3>
-<p>No, but you can mark and entry as “out of scope” if you think it is not relevant to the project.</p>
-<h3>WHAT IS A SURT?</h3>
-<p>SURT stands for Sort-friendly URI Reordering Transform, and is used by the harvest specialists to understand how URLs are related to one another.  The SURT for a URL shows its domain information from the most general to the most specific, and can identify when a particular domain contains many URLs.  This is what generates the list of “related URLs” that you see in some URL records.  You may find this a useful way of exploring the list of URLs, but you’re not required to use it; this information is intended mainly for use by the harvest specialists.</p>
-<h3>WILL A URL GET CRAWLED IF I DON’T VOTE ON IT?</h3>
-<p>Yes.  All of the known URLs will be harvested to some degree during this project.  Voting a URL “in scope” helps to insure that it will be harvested in-depth, so that the resulting archive will be more complete.</p>
-<h3>HOW IS THE NOMINATION SCORE CALCULATED?</h3>
-<p>The initial URL list is compiled from several different sources.  A URL will get one point for each source list it appeared on.  The URL will get one point every time it is voted “in scope” and will lose a point every time it is voted “out of scope”. </p>
-<h3>CAN I VOTE ON A URL MULTIPLE TIMES?</h3>
-<p>You cannot mark a URL “in scope” repeatedly to increase its score, but you can change your vote in order to correct a mistake if needed. </p>
-<h3>WHAT IF I HAVE A QUESTION?</h3>
-<p>If you have questions about the nomination tool, please contact <a href="mailto:mark.phillips@unt.edu">mark.phillips@unt.edu</a></p>
-
 
+    <div class="panel-group">
+        <div class="panel panel-primary">
+            <div class="panel-heading">
+                <h4 class="panel-title">Nomination Tool: How to Contribute</h4>
+            </div>
+            <div class="panel-body">
+                <h5><b>Overview:</b></h5>
+                <p>The purpose of this nomination tool is to allow subject experts to:</p>
+                <ul>
+                    <li>Flag which URLs are “in scope” and most critical to harvest</li>
+                    <li>Add new relevant URLs not appearing in the list yet</li>
+                    <li>Flag which URLs are either wrong or “out of scope” and not relevant to the harvest</li>
+                    <li>Supply basic metadata about a URL, such as site title or description</li>
+                </ul>
+                <p>
+                    Each URL in the list will have a nomination score.  Every time a subject expert
+                    marks a URL “in scope”, the score will rise.  Mark a URL as “out of scope” and
+                    the score will fall.  The nomination score will be used to weight the depth and
+                    frequency of harvests, to insure that critical sites are captured completely.
+                </p>
+                <br>
```

Comment:
If the goal here is to create additional white space following a logical chunk of information, you should encapsulate the logical chunk into a `<div>` or other block element and then use `margin-bottom` on the element.

It looks like that is what you want to do surrounding your `<h5>` heading and the paragraphs that follow within that category.